pkg/server/router/webhook.go

```diff
@@ -0,0 +1,206 @@
+package router
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+
+	"github.com/tbd54566975/ssi-service/pkg/server/framework"
+	svcframework "github.com/tbd54566975/ssi-service/pkg/service/framework"
+	"github.com/tbd54566975/ssi-service/pkg/service/webhook"
+)
+
+type WebhookRouter struct {
+	service *webhook.Service
+}
+
+func NewWebhookRouter(s svcframework.Service) (*WebhookRouter, error) {
+	if s == nil {
+		return nil, errors.New("service cannot be nil")
+	}
+	webhookService, ok := s.(*webhook.Service)
+	if !ok {
+		return nil, fmt.Errorf("could not create webhook router with service type: %s", s.Type())
+	}
+	return &WebhookRouter{service: webhookService}, nil
+}
+
+type CreateWebhookRequest struct {
+	// The noun (entity) for the new webhook.eg: Credential
+	Noun webhook.Noun `json:"noun" validate:"required"`
+	// The verb for the new webhook.eg: Create
+	Verb webhook.Verb `json:"verb" validate:"required"`
```

Comment:
Since Noun's and Verb's can only be from a set of possible values, I think it's important to clarify this in the docs. 

Additionally, where do you think the explanation of when each of the Noun.Verb combinations would happen go? (consider a developer reading the webhook documentation for the first time).